src/AudioManagerImpl.js

```diff
@@ -76,27 +39,25 @@ export function getNoteForState (characterState: CharacterState) : string {
     return noteName;
 }
 
-export default class AudioManagerImpl
-implements AudioManager {
+export default class AudioManager {
```

Comment:
Why the change here to remove the `implements AudioManager`?

Reply:
Pretty sure I messed it up when removing the type we got rid of.  It's fixed now.